pkg/config/database.go

```diff
@@ -42,6 +42,7 @@ func (d *Database) Open(logger *logging.Logger) (*icingadb.DB, error) {
 
 		config.User = d.User
 		config.Passwd = d.Password
+		config.Logger = icingadb.MysqlFuncLogger(logger.Debug)
```

Comment:
I like the [documentation](https://pkg.go.dev/github.com/go-sql-driver/mysql@v1.8.1#Config) :smiling_face_with_tear:
```
Logger               Logger            // Logger
```